html/comparison.html.haml

```diff
@@ -15,6 +15,20 @@
     %script{:src => "assets/transition.js"}
     %script{:src => "assets/collapse.js"}
 
+    %script#alert{:type => "text/ng-template"}
+      #alert_container.scope{"ng-show" => "diff.meta.uninspected"}
+        .row
+          .col-xs-1
+            %span.dismiss{:title => "Collapse/Expand"}
+        .row
+          .row
+            .col-xs-1
+            .col-xs-7
+              Couldn't compare <strong>'{{diff.meta.uninspected[diff.meta.description_a].join(", ")}}'</strong> because they are not present in description <strong>'{{diff.meta.description_a}}'</strong>
+          .row
+            .col-xs-1
+            .col-xs-7
+              Couldn't compare <strong>'{{diff.meta.uninspected[diff.meta.description_b].join(", ")}}'</strong> because they are not present in description <strong>'{{diff.meta.description_b}}'</strong>
```

Comment:
Maybe we can tweak the appearance a bit, e.g. like so:

![image](https://cloud.githubusercontent.com/assets/203813/8614429/ef84c4fc-26e9-11e5-8846-136beb14a711.png)